regions/shapes/circle.py

```diff
@@ -43,8 +44,16 @@ def to_shapely(self):
         return self.center.to_shapely().buffer(self.radius)
 
     def to_sky(self, mywcs, mode='local', tolerance=None):
-        # TODO: needs to be implemented
-        raise NotImplementedError("")
+        if mode != 'local':
+            raise NotImplementedError()
```

Comment:
You don't need to call the constructor, i.e. just always type `raise NotImplementedError` instead of `raise NotImplementedError()` and save two characters (which will be many characters over your lifetime)!